iceberg/iceberg-handler/src/test/queries/positive/iceberg_major_compaction_partition_evolution_ordered.q

```diff
@@ -34,27 +34,30 @@ tblproperties ('format-version'='2');
 
 insert into ice_orc VALUES 
 ('fn1','ln1', 1, 10, 100),
-('fn2','ln2', 1, 10, 100),
+('fn2','ln2', 1, 10, 100);
+
+insert into ice_orc VALUES 
 ('fn3','ln3', 1, 11, 100),
 ('fn4','ln4', 1, 11, 100);
 
 alter table ice_orc set partition spec(dept_id);
 
 insert into ice_orc VALUES 
 ('fn5','ln5', 2, 20, 101),
-('fn6','ln6', 2, 20, 101),
+('fn6','ln6', 2, 20, 101);
+
+insert into ice_orc VALUES 
 ('fn7','ln7', 2, 20, 101),
 ('fn8','ln8', 2, 20, 101);
 
 insert into ice_orc VALUES 
 ('fn9', 'ln9',  3, 20, 101),
-('fn10','ln10', 3, 20, 101),
+('fn10','ln10', 3, 20, 101);
+
+insert into ice_orc VALUES 
 ('fn11','ln11', 3, 20, 101),
 ('fn12','ln12', 3, 20, 101);
 
-select * from ice_orc where company_id = 100;
-select * from ice_orc where dept_id = 2;
```

Comment:
Why did you delete those selects?

Reply:
Because with the other changes done to the q file in the PR the order of the records in the selects is not fixed and changes randomly in different executions of the test.

Reply:
how about adding `-- SORT_QUERY_RESULTS` ?

Reply:
This q-test tests if records get sorted after executing compaction with sort clause.
I removed `-- SORT_QUERY_RESULTS` intentionally, otherwise it won't be possible to test if records are sorted after compaction.

Reply:
Why is there an impact of this PR to change the order of the result of this query?

Reply:
Because of changes that I did on the lines 35-60 in iceberg_major_compaction_partition_evolution_ordered.q